### DISCUSSION TITLE
How to add auto-save plugin in CosmicNvim?

### DISCUSSION THREAD

**Discussion by Juninhoww2** [CosmicNvim/CosmicNvim#54] [Q&A] [answered]
I wanna try install plugin in CosmicNvim. Any help are welcome

github:https://github.com/Pocco81/AutoSave.nvim

**Comment by mattleong** [comment, marked as answer]
@Juninhoww2 You can do so by adding it to your config file. See [here](https://github.com/CosmicNvim/CosmicNvim/tree/main/lua/cosmic/config/examples) for examples

**Comment by Juninhoww2** [comment]
Thanks!
